tsconfig.app.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./src/app.tsx","./src/vite-env.d.ts","./src/components/alert.tsx","./src/components/button.tsx","./src/components/controls.tsx","./src/components/dropdown.tsx","./src/components/icon.tsx","./src/components/link.tsx","./src/components/loading.tsx","./src/components/map.tsx","./src/components/meeting.tsx","./src/components/table.tsx","./src/components/title.tsx","./src/components/tsmlui.tsx","./src/components/index.ts","./src/helpers/analytics.ts","./src/helpers/calculate-distances.ts","./src/helpers/filter-meeting-data.ts","./src/helpers/flatten-and-sort-indexes.ts","./src/helpers/format-address.ts","./src/helpers/format-array.ts","./src/helpers/format-conference-provider.ts","./src/helpers/format-directions-url.ts","./src/helpers/format-feedback-email.ts","./src/helpers/format-ics.ts","./src/helpers/format-slug.ts","./src/helpers/format-string.ts","./src/helpers/format-url.ts","./src/helpers/get-index-by-key.ts","./src/helpers/index.ts","./src/helpers/load-meeting-data.ts","./src/helpers/query-string.ts","./src/helpers/settings.ts","./src/helpers/translate-google-sheet.ts","./src/helpers/user-agent.ts","./src/i18n/en.ts","./src/i18n/es.ts","./src/i18n/fr.ts","./src/i18n/index.ts","./src/i18n/ja.ts","./src/i18n/nl.ts","./src/i18n/pt.ts","./src/i18n/sk.ts","./src/i18n/sv.ts","./src/styles/alert.ts","./src/styles/button.ts","./src/styles/controls.ts","./src/styles/dropdown.ts","./src/styles/global.ts","./src/styles/index.ts","./src/styles/loading.ts","./src/styles/map.ts","./src/styles/meeting.ts","./src/styles/table.ts","./src/styles/variables.ts","./src/types/jsondata.ts","./src/types/meeting.ts","./src/types/metatype.ts","./src/types/settings.ts","./src/types/state.ts","./src/types/translation.ts","./src/types/index.ts"],"version":"5.6.2"}
```

Comment:
When I build this locally, this file changes - `/src/components/alert.tsx` becomes `/src/components/Alert.tsx`, etc. Case of files match local file paths. Not sure if this is a difference between iOS and Linux.

Reply:
huh, it might be mac vs linux! we can keep an eye on that - i'd kind of prefer this file to use the correct casing